recipes/edyn/0.x.x/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conans import ConanFile, CMake, tools
+
+
+class EdynConan(ConanFile):
+    name = "edyn"
+    license = "MIT"
+    homepage = "https://github.com/xissburg/edyn"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Edyn is a real-time physics engine organized as an ECS. "
+    topics = ("game-development", "physics-engine", "ecs",
+              "entity-component-system", "entt")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package", "cmake_paths"
+    exports_sources = "src/**"
+
+    @property
+    def _source_subfolder(self):
+        return "edyn"
+
+    def configure(self):
+        minimal_cpp_standard = "17"
+
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, minimal_cpp_standard)
+
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+        if self.version == "0.1.0":
+            self.requires("entt/3.5.2")
+        elif self.version == "0.2.0":
+            self.requires("entt/3.8.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder)
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses")
+        for pattern in ("*.h", "*.hpp"):
+            self.copy(pattern, dst="include", src="include")
```

Comment:
Since you're the author of edyn, I advise you to add install support to the cmake script.
It's a matter of a few calls to [`install`](https://cmake.org/cmake/help/latest/command/install.html).

e.g. (untested)
```cmake
include(GNUInstallDirs)
install(TARGETS Edyn EXPORT EdynTargets
    ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
    LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
    RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
)
install(DIRECTORY include/edyn src/edyn
    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
    FILES_MATCHING
        PATTERN "*.h"
        PATTERN "*.hpp"
)
install(EXPORT EdynTargets
    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Edyn"
    NAMESPACE "Edyn::"
    FILE "Edyn-config.cmake"
)
```

Reply:
I am not sure into which CMake script this would be added to.

Reply:
This is for the cmake script on the Edyn repo.

Reply:
@xissburg 
I'm working on preparing a pr on your repo to revamp the cmake + conan script.

Reply:
See https://github.com/xissburg/edyn/pull/57